var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -175,7 +176,7 @@ class Trilinos(CMakePackage):
 
     resource(name='dtk',
              git='https://github.com/ornl-cees/DataTransferKit.git',
-             tag='master',
+             commit='4fe4d9d56cfd4f8a61f392b81d8efd0e389ee764',  # branch dtk-3.0
```

Comment:
I don't have a strong preference, but we could tag dtk for this release to clean this up.

Reply:
Thanks, did not know that. Then, there is little point of doing that.